test/acceptance/tests.py

```diff
@@ -472,12 +488,7 @@ def test_staff_area_buttons(self):
 
     @retry()
     @attr('acceptance')
-    @ddt.data(
-        ("staff-tools", "MANAGE INDIVIDUAL LEARNERS"),
-        ("staff-info", "VIEW ASSIGNMENT STATISTICS"),
-    )
-    @ddt.unpack
-    def test_staff_area_panel(self, panel_name, button_label):
```

Comment:
By moving these from a ddt test into the test as a loop, we save the extra `auto_auth_page.visit()`. I can revert this part if anyone feels strongly about it - I think it makes sense to test them together.